src/Math/Angle.hpp

```diff
@@ -106,7 +106,7 @@ class Angle
     }
   };
 
-  constexpr Angle(DMS dms) noexcept:Angle(Degrees(dms.ToFloat())) {}
+  constexpr Angle(const DMS &dms) noexcept : Angle(Degrees(dms.ToFloat())) {}
```

Comment:
What's the point?

Reply:
The value that's passed, isn't modified in the function. Why would you reinstantiate it?

Reply:
I asked first :-)
Clarification: I'm not asking about adding `const` - adding that would be okay (though all it would really do is add noise with no actual use, because it's trivial to see nobody modifies the value); my question was about the `&`, i.e. why it's useful to make this a reference. You proposed to make it a reference, and I ask why. You can't reply with another question, it's your proposal and it's up to you to explain it.

Reply:
I want to pass it by reference so the value isn't copied every time, wasting memory and adding additional operations. 
The const is a safety so nothing ever in that function touches the referenced value, as there are potential other consumers for that value.

Its also marking the function clearly as not modifying of any arguments, which makes it easier to understand when reading / understanding the code. 

While one could argue that this is a small object and the indirection overhead may be a disadvantage, the compiler optimization such as parameter passing will decide on that. 

Like this its clear that we want the place in the code to be optimized.